### PR TITLE
Retrieve size from ended epochs

### PR DESCRIPTION
**Pull request by onmax**
Also added system to allow validators to submit a PR to add their info

### CODE REVIEW

File: packages/nimiq-validators-score/src/utils.ts [hunk 1]
```diff
@@ -5,42 +5,52 @@ interface GetRangeOptions {
   // The last epoch number that we will consider. Default to the last finished epoch.
   toEpochIndex?: number
   // The amount of milliseconds we want to consider. Default to 9 months.
```
Comment (Albermonte) [inline packages/nimiq-validators-score/src/utils.ts]: Duplicated comment

File: server/database/schema.ts [hunk 2]
```diff
@@ -1,32 +1,44 @@
-import { integer, primaryKey, real, sqliteTable, text } from 'drizzle-orm/sqlite-core'
+import { index, integer, primaryKey, real, sqliteTable, text, uniqueIndex } from 'drizzle-orm/sqlite-core'
+
+// TODO
+// Is delete on cascade ok?
 
 export const validators = sqliteTable('validators', {
   id: integer('id').primaryKey({ autoIncrement: true, onConflict: 'replace' }),
-  name: text('name').default('Unknown validator'),
-  address: text('address').notNull().unique(),
```
Comment (Albermonte) [inline server/database/schema.ts:6]: Duplicated validators are allowed?

File: server/utils/activities.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  const missingEpochs = []
+  for (let i = range.fromEpoch; i <= range.toEpoch; i++) {
+    if (!existingEpochs.includes(i))
+      missingEpochs.push(i)
+  }
+  return missingEpochs
+}
+
+/**
+ * We loop over all the pairs activities/epochBlockNumber and store the validator.
```
Comment (Albermonte) [inline server/utils/activities.ts]: `the validator` or `the validator activities`?

File: server/database/schema.ts (hunk 2 above)
Comment (onmax) [reply server/database/schema.ts:6]: oops

File: server/utils/activities.ts (hunk 3 above)
Comment (onmax) [reply server/utils/activities.ts]: yeah better.

Btw, suggestion to rename activities? i don't really like it

**Pull request merged**